initiator.go

```diff
@@ -682,43 +701,46 @@ func (s *IscsiTargetSession) Login(hostname string) error {
 
 // MountIscsi connects to the given iscsi target and mounts it, returning the
 // device name on success
-func MountIscsi(opts ...Option) (string, error) {
+func MountIscsi(opts ...Option) ([]string, error) {
 	netlink, err := ConnectNetlink()
 	if err != nil {
-		return "", err
+		return nil, err
 	}
 	// We use local hostname to identify ourselves to the target
 	hostname, err := os.Hostname()
```

Comment:
i think now that the initiator is a parameter, the hostname here is unnecessary, and you can remove it from Login as well?